tests/test_libontology.py

```diff
@@ -81,6 +86,74 @@ def io_json_file(io_data) -> Generator[Path, None, None]:
         yield Path(f.name)
 
 
+@pytest.fixture
+def molecule():
+    return Molecule(None)
```

Comment:
Strange this is not raising a type error, but I see your point, just add a  `# type: ignore`